app/src/main/java/org/oppia/app/help/HelpActivityPresenter.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.app.help
+
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.drawerlayout.widget.DrawerLayout
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.app.drawer.NavigationDrawerFragment
+import javax.inject.Inject
+
+/** The presenter for [HelpActivity]. */
+@ActivityScope
+class HelpActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  private var toolbar: Toolbar? = null
+  private var navigationDrawerFragment: NavigationDrawerFragment? = null
+
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.home_activity)
+    setUpNavigationDrawer(activity.getString(R.string.menu_help))
```

Comment:
Does this work with rotations? I seem to remember @jamesxu0 having some difficulties with the order in which the toolbar is initialize (as I recall, it's initialized after onCreate post-rotation).

Reply:
@BenHenning I have tested on rotation as well and was working as expected.
@jamesxu0 May i know what was the difficulty you were facing and please let me know thersteps to reproduce the same issue...

Reply:
As I recall, I think the issue was that the toolbar was null after a configuration change in a downstream fragment that tried referencing it (due to the fragment being retained upon a configuration change and the toolbar needing to be recreated). Maybe the issue doesn't exist here for another reason--it's not entirely clear to me but I also don't fully remember the original issue that James hit.